docs/api/materials/ShadowMaterial.html

```diff
@@ -34,7 +34,11 @@
 
 		<h2>Constructor</h2>
 
-		<h3>[name]( )</h3>
+		<h3>[name]( [page:Object parameters] )</h3>
+		<div>
+			[page:Object parameters] - (optional) an object with one or more properties defining the material's appearance.
+			Any property of the material (including any property inherited from [page:Material] and [page:ShaderMaterial]) can be passed in here.<br /><br />
```

Comment:
What is the best way to handle this? Listing all properties that are supported?  Or just remove the last sentence.

BTW: The sentence "_Any property of the material (including any property inherited from [page:Material]) can be passed in here._" exists in nearly all material doc pages. Seems there are some more inconsistencies since `MeshDepthMaterial` or `MeshNormalMaterial` also don't support all properties.